embark/workers/tests.py

```diff
@@ -1 +1,77 @@
 # Create your tests here.
+__copyright__ = 'Copyright 2021-2025 Siemens Energy AG, Copyright 2021-2025 The AMOS Projects'
+__author__ = 'Benedikt Kuehne, ashiven'
+__license__ = 'MIT'
+
+import secrets
+from django.test import TestCase, Client
+
+from workers.models import Worker, Configuration
+from workers.orchestrator import WorkerOrchestrator
+from users.models import User
+
+
+class TestOrchestrator(TestCase):
+    def setUp(self):
+        user = User.objects.create(username='test123')
+        user.set_password('12345')
+        user.api_key = secrets.token_urlsafe(32)
+        user.save()
+        self.client = Client()
+        Configuration.objects.create(user=user, name='test_config', ssh_user='test_user', ssh_password='test_password', ip_range='192.111.111/32')  # nosec
+        test_worker1 = Worker.objects.create(name='test_worker1', ip_address='192.111.111', system_info={}, reachable=True)
```

Comment:
`worker1` can also be saved like so, so `Worker.objects.get()` doesn't need to be called each time: 
```suggestion
        self.worker1 = Worker.objects.create(name='test_worker1', ip_address='192.111.111.111', system_info={}, reachable=True)
```